### DISCUSSION TITLE
higher contrast image

### DISCUSSION THREAD

**Discussion by hugocomunica** [iina/iina#3990] [General]
why do I get a higher contrast image than with other players? 
I have tried QuickTime and vlc and in both I get a less contrasted picture by default without applying any correction of course. 
this happens with any type of file you use mp4, prores, etc. 
<img width="1060" alt="Captura de Pantalla 2022-10-15 a las 23 00 59" src="https://user-images.githubusercontent.com/14318509/196007499-63a5b2f2-d8f3-464c-80d0-c2da5a4a857f.png">

**Comment by hugocomunica** [comment]
the image above is the QuickTime player 
the bottom image is the IINA player
I forgot to specify it in the post

**Reply by Bamgie1** [reply]
I think it has something to do with HDR. This thing happened in 1.3 when they added HDR support. Now every non-HDR video is just darker. Watching a movie is impossible on default settings, every night scene is like a complete darkness. So I got back to the 1.2 version. I just tried 1.3.1, hoping they fixed it, but they didn't. So I'm staying on 1.2, where everything is fine. Except for subtitle searcher, it is not working, which is a bummer.
